### ISSUE TITLE
Does it work on mac?

### ISSUE THREAD

**Issue by whosfry** [deathsec/instagram-py#21]
**Select your issue type:** (check at least one)
- [ ] Bug
- [X ] Question
- [ ] Suggestion
- [ ] Other (please describe):

Hello! Does it work on Mac or just Linux??

**Comment by bird** [comment]
Yes, it works on macOS. Setup just varies a bit as the default tor port is different, at least in my case (9150 vs 9050)

**Comment by arminfaaa** [comment]
> Yes, it works on macOS. Setup just varies a bit as the default tor port is different, at least in my case (9150 vs 9050)

hey dear can you please explain how do you run this script in macos.i will appreciate.
